apps/src/aichat/views/modelCustomization/FieldLabel.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {WithTooltip} from '@cdo/apps/componentLibrary/tooltip';
+import {BodyThreeText, StrongText} from '@cdo/apps/componentLibrary/typography';
+
+import styles from '../model-customization-workspace.module.scss';
+
+interface FieldLabelProps {
+  id: string;
+  label: string;
+  tooltipText: string;
+}
+
+const FieldLabel: React.FunctionComponent<FieldLabelProps> = ({
+  id,
+  label,
+  tooltipText,
+}) => {
+  return (
+    <div className={styles.fieldLabelContainer}>
+      <label htmlFor={id}>
+        <BodyThreeText className={styles.fieldLabel}>
+          <StrongText>{label}</StrongText>
+        </BodyThreeText>
+      </label>
+      <WithTooltip
+        tooltipProps={{
+          text: tooltipText,
+          size: 's',
+          tooltipId: `${id}-tooltip`,
+          direction: 'onRight',
+          className: styles.tooltip,
+        }}
+      >
+        <div>
```

Comment:
Could we make these buttons so that they are clickable and tab-navigable?

Reply:
Good call, done!

Reply:
Awesome! Just noting that when keyboard-navigating, the tooltip is immediately displayed when the user navigates to the icon so being clickable is not necessary - this seems right to me! Just confirming with  @samantha-code.

Reply:
Yeah I think that also sounds right to me? If we want the tooltip to appear only after clicking/pressing enter, that's separate functionality we'd have to implement in the Tooltip component.